api/weather-data.js

```diff
@@ -34,3 +34,44 @@ function getWeatherData(callback) {
     }
   });
 }
+
+module.exports.getWeatherDataByCoords = getWeatherDataByCoords;
+
+function getWeatherDataByCoords(lat, long, callback) {
+  var url = 'http://api.met.no/weatherapi/locationforecast/1.9/?lat=' + lat + ';lon=' + long;
+  request(url, function(error, response, body) {
+    if (error) {
+      callback(error);
+    } else if (response.statusCode != 200) {
+      callback(new Error(response.statusCode));
+    } else {
+      var xmlDoc = libxmljs.parseXml(body);
+
+      var currentTimePeriod = xmlDoc.get('//product/time/location');
+
+      var temperature = currentTimePeriod.get('temperature/@value').value();
+      var windSpeed = currentTimePeriod.get('windSpeed/@mps').value();
+      var effectiveTemperature = 13.12 +
+                                 0.6215 * temperature -
+                                 11.37 * Math.pow(windSpeed, 0.16) +
+                                 0.3965 * temperature * Math.pow(windSpeed, 0.16);
+
+      // Symbols are weird in the met.no API
+      // The first time data-point doesn't seem to have symbol data,
+      // but the next one usually does.
+      var symbol = xmlDoc.get('//product/time/location/symbol/@number').value();
```

Comment:
Will this query get a reference to another time-element than the one fetched in `currentTimePeriod`?
That is, does the get method of `xmlDoc` pick a new element each time it's evaluated?